emission/analysis/classification/inference/labels/pipeline.py

```diff
@@ -53,11 +53,11 @@ def placeholder_predictor_1(trip):
 
 # This third scenario provides labels designed to test the soundness and resilience of
 # the client-side inference processing algorithms.
-trip_n = 0  # ugly use of globals for testing only
 def placeholder_predictor_2(trip):
-    global trip_n  # ugly use of globals for testing only
-    trip_n %= 6
-    trip_n += 1
+    # Hardcoded to match "test_july_22" -- clearly, this is just for testing
+    timestamp2index = {494: 5, 565: 4, 795: 3, 805: 2, 880: 1, 960: 0}
```

Comment:
so these represent the timestamps of the trips in `test_july_22`? Does this still work for your ongoing testing?

Reply:
It does — so far I've only been testing with `test_july_22`. This was the simplest way I could think of to remove `trip_n` and make the prediction a pure function of the trip, but if it's not flexible enough for me later on, I'll find a more complicated way.